installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc

```diff
@@ -31,11 +31,19 @@ include::modules/installing-ocp-agent-inputs.adoc[leveloffset=+2]
 
 You can create additional manifests to further configure your cluster beyond the configurations available in the `install-config.yaml` and `agent-config.yaml` files.
 
-If you do not want to make additional configurations, proceed to xref:../../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc#installing-ocp-agent-boot_installing-with-agent-based-installer[Creating and booting the agent image].
+If you do not want to make additional configurations, proceed to the next section.
```

Comment:
Same comment I made below about the module title stating "Optional" I am not sure the sentence is needed.